jest.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
What about moving the config to `package.json`? That saves another file (2?) in the repo: https://jestjs.io/docs/configuration

Reply:
Oh that's a good idea, I'll do that